### ISSUE TITLE
[a11y]:  activity accessibility violations in data table component 

### ISSUE THREAD

**Issue by jingx-xue** [carbon-design-system/carbon#10459]
### Package

carbon-components-react

### Browser

Chrome

### Operating System

Windows

### Package version

10.43.0

### React version

16.8.6

### Automated testing tool and ruleset

1.3.1 Info and Relationships

### Description

![image](https://user-images.githubusercontent.com/48215968/149858687-ee1f6956-e36f-436c-bf20-88a75a31df9c.png)

### WCAG 2.1 Violation

1.3.1 Info and Relationships

### CodeSandbox example

https://codesandbox.io/s/gallant-lumiere-7q18y

### Steps to reproduce

Open Sandbox example
Run accessibility tool

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by joshblack** [comment]
Hi there @jingx-xue! 👋 

When you get a chance, can you fill out the CodeSandbox link with an example that demonstrates the violation that you're running into? Thanks!

**Comment by jingx-xue** [comment]
@joshblack Hi Josh, seems the css style sheet is lost in the sandbox, can you help me with it?
![image](https://user-images.githubusercontent.com/48215968/167470431-5e782c0d-918e-4e4b-a757-1172fda4cfa9.png)

**Comment by tay1orjones** [comment]
I believe this was addressed over in: https://github.com/carbon-design-system/carbon/pull/10547 The storybook story should now be up-to-date for this use-case and teams should implement those changes on their end so that the AVT violation goes away
